docs/before/btx.md

```diff
@@ -0,0 +1,71 @@
+
+!!! note "Quicklink"
+    - **btx** - [btx Github repository][1]
+    - **documentation** - [btx code documentation][2]
+
+# Introduction to *btx*
+
+*btx* is a resource for analyzing experiments performed at LCLS, both during and after beamtime. To facilitate rapid analysis, common data processing steps have been organized into tasks that can be easily accessed from the eLog or command-line without users having to write new code. This is a primarily Python-based package to which contributions are welcome, as *btx* is intended as a library to serve the user community at LCLS. Currently most tasks are geared toward SFX data analysis, but we intend to expand the scope to encompass other experiments, such as SPI and SAXS/WAXS.
+
+## Usage
+
+### Preparing the configuration file
+
+To run *btx*, we first need to generate a configuration file that lists the desired tasks and defines their corresponding parameters. This file follows yaml format; a partial example file is shown below:
```

Comment:
Not exactly related to the page _per se_ but maybe it is worth writing a script which will generate an example with sensible default values. So something like this from the command line:

```
$> generate_yaml <exp_name> <det_type>
```

which outputs the yaml in a hierarchy like this with prepopulated setup section and basic tasks:
```
${SIT_PSDM_DATA}/{instrument}/{exp}/scratch/btx/
└──yaml/
    └── default_config.yaml
```